bintool/mbedtls_wrapper.c

```diff
@@ -187,7 +191,11 @@ void mb_sign_sha256(const uint8_t *entropy, size_t entropy_size, const message_d
 
     if ((ret = mbedtls_ecdsa_write_signature(&ctx_sign, MBEDTLS_MD_SHA256,
                                              m->bytes, sizeof(m->bytes),
-                                             out->der, &out->der_len,
+                                             out->der,
+                                             #if MBEDTLS_VERSION_MAJOR >= 3
```

Comment:
Only a minor nitpick, but would it be "safer" to stick to just one style of `#if`, rather than mixing `#if MBEDTLS_VERSION_MAJOR == 2` and `#if MBEDTLS_VERSION_MAJOR >= 3` ?

Reply:
I've changed them all to be >= 3